test/doc/traits/callable_specs.cpp

```diff
@@ -0,0 +1,56 @@
+#include "eve/detail/raberu.hpp"
+#include <iostream>
+#include <type_traits>
+#include <eve/traits/overload.hpp>
+#include <eve/wide.hpp>
+
+namespace eve
+{
+  // Defines two RABERU flags
+  using namespace rbr::literals;
+  inline constexpr auto precise = "precise"_fl;
+  inline constexpr auto scale   = "scale"_fl;
```

Comment:
let's use structs, no?

Reply:
I wanted to keep the example focused on the support thing, the fact we use the fast API of RABERU is a detail.

Reply:
literals are much more confusing then just a struct